homeassistant/components/apprise/notify.py

```diff
@@ -0,0 +1,71 @@
+"""Apprise platform for notify component."""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+from homeassistant.components.notify import (
+    ATTR_TITLE,
+    ATTR_TITLE_DEFAULT,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_FILE = "config"
+CONF_URL = "url"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Optional(CONF_URL): vol.All(cv.ensure_list, [str]),
+        vol.Optional(CONF_FILE): cv.string
+    }
+)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Apprise notification service."""
+    from apprise import Apprise
+    from apprise import AppriseConfig
```

Comment:
could be combined
`from apprise import Apprise, Appriseconfig`

Reply:
Thanks; just pushed an update to accommodate this now.